settings.gradle.kts

```diff
@@ -1,5 +1,7 @@
 rootProject.name = "hmpps-spring-boot-kotlin"
 include("hmpps-kotlin-spring-boot-autoconfigure")
+include("hmpps-kotlin-spring-boot-test-autoconfigure")
 include("hmpps-kotlin-spring-boot-starter")
+include("hmpps-kotlin-spring-boot-starter-test")
```

Comment:
this seems a bit weird calling one `starter-test` and the other `test-autoconfigure`, but I've followed the same convention as spring e.g. https://mvnrepository.com/artifact/org.springframework.boot/spring-boot-starter-test and also https://mvnrepository.com/artifact/org.springframework.boot/spring-boot-test-autoconfigure